hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterRpc.java

```diff
@@ -1757,7 +1757,7 @@ public void testRBFMetricsMethodsRelayOnStateStore() {
     // These methods relays on
     // {@link RBFMetrics#getActiveNamenodeRegistration()}
     assertEquals("{}", metrics.getNameservices());
-    assertEquals(0, metrics.getNumLiveNodes());
+    assertEquals(NUM_DNS * 2, metrics.getNumLiveNodes());
```

Comment:
Why now this is like this?

Reply:
GetNumLiveNodes used to obtain dn information through StateStore, and now it obtains it through RouterRpcServer#getCachedDatanodeReport. Now this test is not applicable in StateStore, I deleted it.